experimental/saiportextensions.h

```diff
@@ -52,6 +52,18 @@ typedef enum _sai_port_stat_extensions_t
 {
     SAI_PORT_STAT_EXTENSIONS_RANGE_START = SAI_PORT_STAT_END,
 
+    /** DASH port LB_FAST_PATH_ICMP_IN_BYTES stat count */
+    SAI_PORT_STAT_LB_FAST_PATH_ICMP_IN_BYTES = SAI_PORT_STAT_EXTENSIONS_RANGE_START,
+
+    /** DASH port LB_FAST_PATH_ICMP_IN_PACKETS stat count */
+    SAI_PORT_STAT_LB_FAST_PATH_ICMP_IN_PACKETS,
+
+    /** DASH port LB_FAST_PATH_ENI_MISS_BYTES stat count */
+    SAI_PORT_STAT_LB_FAST_PATH_ENI_MISS_BYTES,
```

Comment:
What do ENI miss bytes indicate?

Reply:
got it from here: https://github.com/sonic-net/DASH/blob/main/documentation/load-bal-service/fast-path-icmp-flow-redirection.md#port-stats-attributes

Reply:
Please confirm if these are DPU level counters. e.g., if a packet if given to a DPU due to PA match, but no ENI match happened once it hits DPU and packet is dropped?

Reply:
yes, these are DPU level counters. and yes, it is designed to capture the exact case you are referring to. In this case, nothing will show up on ENI counters. With this counter, we will know what there are valid fast path packets gets dropped, because ENI cannot be found.